src/clearcase.ts

```diff
@@ -292,16 +292,25 @@ export class ClearCase {
   public async findCheckouts(): Promise<string[]> {
     let lscoArgTmpl = this.configHandler.configuration.FindCheckoutsCommand.Value;
     let results: string[] = [];
+	let resNew: string[] = [];
```

Comment:
Can you please check the indentation of the new lines? It doesn't match with the current one.

Reply:
Yep, used tabs instead of spaces... Will update this.

Reply:
Ok thanks, I'm going to merge this as soon as you fix the indentation.